demos/human_pose_estimation_demo/include/human_pose_estimation_demo.hpp

```diff
@@ -13,8 +13,11 @@ static const char help_message[] = "Print a usage message.";
 /// @brief Message for video argument
 static const char video_message[] = "Required. Path to a video. Default value is \"cam\" to work with camera.";
 
-/// @brief Message for model argument
-static const char human_pose_estimation_model_message[] = "Required. Path to the Human Pose Estimation model (.xml) file.";
+/// @brief Message for openpose model argument
+static const char openpose_model_message[] = "Required. Path to the Human Pose Estimation model (.xml) file.";
```

Comment:
Let's name it as `pose_estimation_model_message`

Reply:
OK, I had finished the modification.